components/brave_rewards/resources/tip/components/tipRedditUser.tsx

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { bindActionCreators, Dispatch } from 'redux'
+import { connect } from 'react-redux'
+
+// Components
+import Banner from './siteBanner'
+import TransientTipOverlay from './transientTipOverlay'
+
+// Utils
+import * as rewardsActions from '../actions/tip_actions'
+import { getLocale } from '../../../../common/locale'
+
+interface Props extends RewardsTip.ComponentProps {
+  url: string
+  publisher: RewardsTip.Publisher
+  redditMetaData: RewardsTip.RedditMetaData
+}
+
+class TipRedditUser extends React.Component<Props, {}> {
```

Comment:
not sure if it's already fixed in your refactor, but don't forget to generalize this as well